language/fr/fleetshortcut.lang

```diff
@@ -1,5 +1,41 @@
 <?php
 
-include("{$_EnginePath}language/en/fleetshortcut.lang");
+$_Lang['Title']                         = 'Cibles personnalisées';
+
+$_Lang['Your_shortcuts']                = 'Vos cibles personnalisées';
+$_Lang['add_new_shortcut']              = 'Ajouter une nouvelle cible';
+$_Lang['select_shortcut']               = 'Sélectionnez la cible';
+$_Lang['edit_shortcut']                 = 'Modifier';
+$_Lang['delete_shortcut']               = 'Supprimer';
+$_Lang['are_you_sure']                  = 'Etes-vous sûr de vouloir supprimer cette cible?';
+$_Lang['goback']                        = 'Retour';
+$_Lang['no_shortcuts']                  = '- Aucune cible disponible -';
+$_Lang['moon_sign']                     = 'L';
+$_Lang['debris_sign']                   = 'CD';
+$_Lang['planet_sign']                   = 'P';
+
+$_Lang['NameTip']                       = 'Le titre de la cible se compose toujours de votre nom personnalisé et du vrai nom de la cible (s\'il existe)';
+$_Lang['Name']                          = 'Custom name <acronym title="'.$_Lang['NameTip'].'">(optional)</acronym>';
+$_Lang['Coordinates']                   = 'Coordonnées';
+$_Lang['Adding_shortcut']               = 'Ajouter une cible personnalisée';
+$_Lang['Editing_shortcut']              = 'Modifier la cible personnalisée';
+$_Lang['Deleting_shortcut']             = 'Modifier la cible personnalisée';
```

Comment:
This is a duplication of the previous translation string. It should be about shortcut removal, not modification.